src/components/diamond.jsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Angle = '30deg';
```

Comment:
In block header component the only contant is `Header` and it is react component. React components must be named with PascalCase